advocacy_docs/supported-open-source/barman/pg-backup-api/01-installation.mdx

```diff
@@ -103,7 +103,30 @@ curl http://localhost:7480/diagnose
    },
    ...
 }
+
 ```
 
+
 You can now query Barman over HTTP, however because Postgres backup API works over plain HTTP it is restricted to serving on localhost only.
 To securely enable remote access follow the instructions in [securing Postgres backup API](02-securing-pg-backup-api).
+
+
+Send a new task to perform a recover by providing metadata like this:
+
+```bash
+curl -X POST http://localhost:7480/servers/<SERVER_NAME>/operations -H "content-type: application/json" -d@payload-pg-backup-api.json
```

Comment:
loalhost?

Reply:
Could be localhost or BARMANSERVER but I did that to follow the rest of the examples in the page